cs/test/core/ConvertTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿namespace UnitTest.Convert
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Reflection;
+    using System.Threading;
+    using Bond;
+    using NUnit.Framework;
+    using Bond.IO.Safe;
+    using Bond.Protocols;
+
+    [global::Bond.Schema]
+    public partial class Decimals
+    {
+        [global::Bond.Id(0), global::Bond.Type(typeof(global::Bond.Tag.blob))]
+        public System.Decimal _dec { get; set; }
```

Comment:
Is `System.Decimal` instead of `decimal` intentional here?

Reply:
Probably not. Corrected.